Code/GraphMol/FMCS/MaximumCommonSubgraph.cpp

```diff
@@ -635,8 +629,8 @@ bool MaximumCommonSubgraph::createSeedFromMCS(size_t newQueryTarget,
                                               Seed& newSeed) {
   Seed mcs;
   mcs.ExcludedBonds.resize(McsIdx.QueryMolecule->getNumBonds());
-  for (size_t i = 0; i < mcs.ExcludedBonds.size(); i++)
-    mcs.ExcludedBonds[i] = false;
+  for (auto && ExcludedBond : mcs.ExcludedBonds)
+    ExcludedBond = false;
```

Comment:
Why two ampersand `auto && ExcludedBond`? Also, should be lower case `excludedBond`